src/ibusunicode.c

```diff
@@ -622,10 +635,11 @@ ibus_unicode_data_load_async_thread (GTask        *task,
 
     retval = ibus_unicode_data_load_with_error (path, source_object, &error);
     g_free (path);
-    if (retval == NULL)
+    if (retval == NULL) {
```

Comment:
Please revert this change. I think it's good not to have braces for the one line indent.